requirements/main.txt

```diff
@@ -1,15 +1,24 @@
 diff_match_patch
 editdistance
+ffmpeg
 hydra-core
 joblib
 librosa>=0.10.0 # specify >=0.10.0 so that librosa.get_duration(path=...) will work
 numpy
 omegaconf
+openpyxl
```

Comment:
Do we need this? I don't think it's used in this PR

Reply:
Done